include/materials/bingham.tcc

```diff
@@ -83,14 +90,17 @@ Eigen::Matrix<double, 6, 1> mpm::Bingham<Tdim>::compute_stress(
 
   // Update pressure
   (*state_vars).at("pressure") +=
-      this->thermodynamic_pressure(ptr->dvolumetric_strain());
+      (compressibility_multiplier_ *
+       this->thermodynamic_pressure(ptr->dvolumetric_strain()));
 
   // Update volumetric and deviatoric stress
   // thermodynamic pressure is from material point
   // stress = -thermodynamic_pressure I + tau, where I is identity matrix or
   // direc_delta in Voigt notation
   const Eigen::Matrix<double, 6, 1> updated_stress =
-      -(*state_vars).at("pressure") * this->dirac_delta() + tau;
+      -(*state_vars).at("pressure") * this->dirac_delta() *
+          compressibility_multiplier_ +
```

Comment:
I don't think you need to multiply here again, given that the `pressure` will be zero.

Reply:
It's not zero. It's computed by the implicit solver. So you need to have it here.

Reply:
I'd leave the multiplier in case there is some initial pressure.

Reply:
I am OK with multiplying then.